README.md

```diff
@@ -120,6 +120,29 @@ Run Container:
 docker run my_custom_cxcli:latest Scan -CxServer http://localhost -CxUser admin@cx -CxPassword password -ProjectName /CxServer/SP/Company/Team/myproject -LocationType folder -LocationPath ./mycode -Log log.log -v
 ```
 
-### License
```

Comment:
@nleach999 why License details were removed from Readme ?

Reply:
The license is in the root of the repo.